resume.tex

```diff
@@ -0,0 +1,123 @@
+
+\input{command.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Now for the actual document:
+
+\begin{document}
+
+\input{header.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%% 
+\begin{ressection}{Employment}
+	\begin{restitledposition}{Nurx / Thirty Madison}{Staff Software Engineer}{April 2022--August 2024}
+		\ressubitem{\textbf{Job Runner} Designed, implemented, and maintained an in-house asynchronous job running
```

Comment:
could add "architected" buzz word. Basically the same as "Designed", but will hit more filters with both

Reply:
not going to add cause that's 4 items in the list and that's too much. I'm going to replace `created` later in the block with `architected`